tools/cli/commands/creategpu.py

```diff
@@ -169,43 +211,53 @@ def run(args, gcloud_beta_compute, gcloud_repos,
     if (not args.zone) and (not args.quiet):
         args.zone = utils.prompt_for_zone(args, gcloud_beta_compute)
     disk_cfg = create.prepare(args, gcloud_beta_compute, gcloud_repos)
+
     print('Creating the instance {0}'.format(args.instance))
     print('\n\nDue to GPU Driver installation, please note that '
```

Comment:
With the new approach, how long does creation take?  Do we still need this admittedly vague message?

Reply:
It still takes a long time; that's a fundamental consequence of downloading/installing the drivers after startup.